sdk/servicebus/Azure.Messaging.ServiceBus/tests/Azure.Messaging.ServiceBus.Tests.csproj

```diff
@@ -25,6 +25,7 @@
     <PackageReference Include="Polly" />
     <PackageReference Include="System.Threading.Tasks.Extensions" />
     <PackageReference Include="System.Memory.Data" />
+    <PackageReference Include="Azure.Storage.Blobs" VersionOverride = "12.9.1" />
```

Comment:
Used this version to leverage the BinaryData APIs. I can update the central version instead if there is no reason not to update it.

Reply:
I have no objection to updating the central version.   I can't speak to what else may be using it, though.